.github/workflows/publish-beta-release.yml

```diff
@@ -70,4 +74,18 @@ jobs:
         run: gh release create v${{ env.VERSION }} **.vsix --title "Pre-Release v${{ env.VERSION }}" --notes-file CHANGELOG.md --prerelease
         working-directory: tmp/saved-extensions
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - id: version
+        run: echo "version=${{ env.VERSION }}" >> $GITHUB_OUTPUT
+
+  slack_notification:
+    if: ${{ always() }}
+    needs: create_git_tag
+    uses: forcedotcom/salesforcedx-vscode/.github/workflows/slack-notification.yml@cristi/slack_notify
```

Comment:
Gotta update this one to match `./.github/workflows/slack-notification.yml`

Reply:
yess, I was waiting to do just one more push because with the previous test I was going to have to merge develop into my branch anyways